presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -169,14 +173,65 @@ following rules:
 For group-based rules to match, users need to be assigned to groups by a
 :doc:`/develop/group-provider`.
 
+.. _schema_rules:
+
+Schema Rules
+------------
+
+These rules allow you to grant ownership of a schema. Having ownership of an
```

Comment:
Let's add the "top to bottom" information here to be consistent with other sections.